client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -186,16 +187,19 @@ export const StagingSiteCard = ( { currentUserId, disabled, siteId, siteOwnerId,
 	const getManageStagingSiteContent = () => {
 		return (
 			<>
-				<p>
-					{ translate( 'Your staging site is available at {{a}}%(stagingSiteName)s{{/a}}.', {
-						args: {
-							stagingSiteName: stagingSite.url,
-						},
-						components: {
-							a: <a href={ stagingSite.url } />,
-						},
-					} ) }
-				</p>
+				<p>{ translate( 'Your staging site is available at:' ) }</p>
+				<div style={ { display: 'flex', alignItems: 'center', marginBottom: 24 } }>
```

Comment:
@katinthehatsite , can we use [Styled components](https://emotion.sh/docs/styled#object-styles) instead of inline style?

We could declare the styled component on the top of the file.
```
const SiteRow = styled.div( {
	display: 'flex',
	alignItems: 'center',
	marginBottom: 24,
	'.site-icon': { marginLeft: '10px', flexShrink: 0 },
} );
```

and use it here:
```suggestion
				<SiteRow>
```

Reply:
Done!